sdk/src/main/java/com/kin/ecosystem/marketplace/view/EarnRecyclerAdapter.java

```diff
@@ -1,10 +1,29 @@
 package com.kin.ecosystem.marketplace.view;
 
+import static com.kin.ecosystem.core.util.DeviceUtils.DensityDpi.XXHDPI;
+
 import com.kin.ecosystem.R;
+import com.kin.ecosystem.core.util.DeviceUtils;
 
 class EarnRecyclerAdapter extends OfferRecyclerAdapter {
 
+    private static final float IMAGE_WIDTH_RATIO = 0.388f;
+    private static final float IMAGE_HEIGHT_RATIO = 0.714f;
+
+	private static final float IMAGE_WIDTH_XX_RES_RATIO = 0.333f;
+	private static final float IMAGE_HEIGHT_XX_RES_RATIO = 0.664f;
+
     EarnRecyclerAdapter() {
         super(R.layout.kinecosystem_earn_recycler_item);
     }
+
+	@Override
+	protected float getImageWidthToScreenRatio() {
```

Comment:
why diffrent method naming  getImageWidthToScreenRatio and getImageHeightRatio (i.e the ToScreen in the width and not in Height)